napari/layers/_points_layer/model.py

```diff
@@ -426,6 +431,32 @@ def get_message(self, coord, value):
             msg = msg + ', index ' + str(value)
         return msg
 
+    def _update_thumbnail(self):
+        """Update thumbnail with current points and colors.
+        """
+        colormapped = np.zeros(self._thumbnail_shape)
+        colormapped[..., 3] = 1
+        if len(self._points_view) > 0:
+            min_vals = [self.range[-2][0], self.range[-1][0]]
+            shape = np.ceil(
+                [
+                    self.range[-2][1] - self.range[-2][0] + 1,
+                    self.range[-1][1] - self.range[-1][0] + 1,
+                ]
+            ).astype(int)
+            zoom_factor = np.divide(self._thumbnail_shape[:2], shape).min()
+            coords = np.floor(
+                (self._points_view - min_vals + 0.5) * zoom_factor
+            ).astype(int)
+            coords = np.clip(
+                coords, 0, np.subtract(self._thumbnail_shape[:2], 1)
+            )
+            for c in coords:
+                colormapped[c[0], c[1], :] = Color(self.face_color).rgba
+        colormapped[..., 3] *= self.opacity
```

Comment:
Do you think it's necessary to modify the opacity of the markers in the thumbnail? Since the thumbnail is for the layer they aren't blended with anything else. I think there would be some opacity/face color/display brightness combinations that would make the markers hard to see and the thumbnail would look black. 

On the other hand, it would be kind of neat to see the opacity change in the thumbnail...

Reply:
The reason I did this was for the labels layer it's less confusing to have the opacity of the thumbnail match the opacity of the layer as the colors themselves then look identical. Otherwise the colors in the thumbnail look very saturated. We could change, but I think it should be standard across all layer types